Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -636,6 +637,26 @@ namespace AssetProcessor
             return;
         }
 
+        if (!m_lfsPointerFileValidator)
```

Comment:
We should consider setting up a command pattern to register validators so we can process additional ones just via registration call. Idea for future PR.